src/components/scenes/MigrateWalletCompletionScene.tsx

```diff
@@ -0,0 +1,253 @@
+import { add, sub } from 'biggystring'
+import { EdgeCurrencyWallet, EdgeSpendInfo, EdgeTransaction } from 'edge-core-js'
+import * as React from 'react'
+import { ActivityIndicator, FlatList, View } from 'react-native'
+import IonIcon from 'react-native-vector-icons/Ionicons'
+
+import { useAsyncEffect } from '../../hooks/useAsyncEffect'
+import { useHandler } from '../../hooks/useHandler'
+import { useWatch } from '../../hooks/useWatch'
+import s from '../../locales/strings'
+import { useSelector } from '../../types/reactRedux'
+import { NavigationProp, RouteProp } from '../../types/routerTypes'
+import { getWalletName } from '../../util/CurrencyWalletHelpers'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { showError } from '../services/AirshipInstance'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { CreateWalletSelectCryptoRow } from '../themed/CreateWalletSelectCryptoRow'
+import { Fade } from '../themed/Fade'
+import { MainButton } from '../themed/MainButton'
+import { SceneHeader } from '../themed/SceneHeader'
+import { MigrateWalletItem } from './MigrateWalletSelectCryptoScene'
+
+interface Props {
+  navigation: NavigationProp<'migrateWalletCompletion'>
+  route: RouteProp<'migrateWalletCompletion'>
+}
+
+interface MigrateWalletTokenItem extends MigrateWalletItem {
+  tokenId: string
+}
+
+const makeSpendSignAndBroadcast = async (wallet: EdgeCurrencyWallet, spendInfo: EdgeSpendInfo): Promise<EdgeTransaction> => {
```

Comment:
Utility functions go to the bottom of the file, right after the component but before the final `export`. That way people reading the file see the component itself first thing (with its props).